pkg/configmanager/load_config.go

```diff
@@ -89,3 +95,70 @@ func yamlFormat(path string) bool {
 	}
 	return false
 }
+
+func checkUpdateFromDyconfig(conf *v2.MOSNConfig) {
+	if _, err := os.Stat(dyconfigPath); os.IsNotExist(err) {
+		return
+	}
+	if cfg := configLoadFunc(dyconfigPath); cfg != nil {
+		// update clusters
+		l := len(conf.ClusterManager.Clusters)
+		for _, cluster := range cfg.ClusterManager.Clusters {
+			found := false
+			for i := 0; i < l; i++ {
+				if cluster.Name == conf.ClusterManager.Clusters[i].Name {
+					log.StartLogger.Debugf("[config] [load dyconfig] cluster %s added\n", cluster.Name)
+					conf.ClusterManager.Clusters[i] = cluster
+					found = true
+					break
+				}
+			}
+			if !found {
+				log.StartLogger.Debugf("[config] [load dyconfig] cluster %s added\n", cluster.Name)
+				conf.ClusterManager.Clusters = append(conf.ClusterManager.Clusters, cluster)
+			}
+		}
+		if len(conf.Servers) == 0 || len(cfg.Servers) == 0 {
```

Comment:
if my config such as [mosn_xds.json](https://github.com/mosn/mosn/blob/master/configs/mosn_xds.json), have no `server` configuration, it will not start succ.